scripts/table_updates/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8
+FROM python:3.9
```

Comment:
Nit: May want to consider just bumping it to Python 3.10 or 3.11 since Python 3.9 is EOL next year.

Reply:
I put 3.9 here to follow what is specified in synapseclient dockerfile. I'm happy to bump it up to a newer version.

Reply:
Hmm, I'll let you decide - I think this works for now.

Reply:
Tested it that the docker image can be built successfully for python 3.11. I will include this change in GEN-1516.